readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

```diff
@@ -132,6 +132,15 @@ if 'html_context' in globals():
 else:
     html_context = context
 
+# Add custom RTD templates
+if using_rtd_theme:
```

Comment:
Overriding the templates only when using our own theme, but what would happen with derivated themes?

They will have the content of the `versions.html` that comes from our `sphinx_rtd_theme` instead of this one.

Reply:
This is a good question. Maybe through some Sphinx internals we can determine if the theme in use extends our theme at some point? This sounds like a hard and error prone thing to determine though.

Ultimately, this might be a really good reason to keep this logic in our theme. The work required to replicate `if READTHEDOCS` is going to be substantial. Even if removing `READTHEDOCS` from our theme is technically correct, leaving it is the *right* thing to do.